index.bs

```diff
@@ -781,35 +681,19 @@ spec: webidl
       Takes an instance of the <a>permission descriptor type</a> and a new or
       existing instance of the <a>permission result type</a>, and updates the
       <a>permission result type</a> instance with the query result. Used by
-      {{Permissions}}' {{Permissions/query()}} method and the <a
+      {{Permissions}}' {{Permissions/query(permissionDesc)}} method and the <a
       href="#PermissionStatus-update">PermissionStatus update steps</a>. If
       unspecified, this defaults to the <a>boolean permission query
       algorithm</a>.
     </dd>
     <dt>
-      A <dfn export>permission request algorithm</dfn>
```

Comment:
We still need this algorithm I think. It's referenced in [mediacapture](https://w3c.github.io/mediacapture-main/getusermedia.html#request-permission).

Reply:
I believe [mediacapture](https://w3c.github.io/mediacapture-main/getusermedia.html#request-permission) intends either https://w3c.github.io/permissions/#request-permission-to-use or https://w3c.github.io/permissions/#prompt-the-user-to-choose. @alvestrand?

Reply:
Right! It's the former. I'll open an issue.